tests/realtikvtest/pipelineddmltest/pipelineddml_test.go

```diff
@@ -127,37 +145,50 @@ func TestPipelinedDMLNegative(t *testing.T) {
 	// not in auto-commit txn
 	tk.MustExec("set session tidb_dml_type = bulk")
 	tk.MustExec("begin")
+	tk.MustQuery("show warnings").CheckContain("Pipelined DML can only be used for auto-commit INSERT, REPLACE, UPDATE or DELETE. Fallback to standard mode")
```

Comment:
Also need to remove this comment

https://github.com/pingcap/tidb/blob/7548df70b1a4e84bd2ad2fa6cb52dfa819f4e5d3/tests/realtikvtest/pipelineddmltest/pipelineddml_test.go#L50